docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -113,6 +113,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `SSH_LISTEN_PORT`: **%(SSH\_PORT)s**: Port for the built-in SSH server.
 - `OFFLINE_MODE`: **false**: Disables use of CDN for static files and Gravatar for profile pictures.
 - `DISABLE_ROUTER_LOG`: **false**: Mute printing of the router log.
+- `MACARON_LOG_LEVEL`: **console**: Sets the macaron log level, with `console` Macaron logs to `os.Stdout`, set this to a valid log level to send it to gitea's default log.
```

Comment:
Is this a log type or a log level?

Reply:
Ok, so things are unfortunately a little more complex. Macaron uses the go log interface and creates it's own go log hence the weird io.Writer shim. Everything Macaron logs internally gets sent to the same log level.

We can just decide that we will log all of this at Info level, then this setting could be simply be console or gitea log?

Another problem is the log level of the router log. Should this also be kept at info level? We can now separate this from Macaron's log so we could simplify the logger here to simply use the gitea logger interface.

Reply:
OK I've moved to a simpler solution whereby the option is to redirect the macaron log or not, with both the router and the other macaron information set at the info level.